crates/bevy_ui/src/lib.rs

```diff
@@ -22,7 +23,10 @@ pub use ui_node::*;
 #[doc(hidden)]
 pub mod prelude {
     #[doc(hidden)]
-    pub use crate::{entity::*, geometry::*, ui_node::*, widget::Button, Interaction, UiScale};
+    pub use crate::{
+        entity::*, geometry::*, layout_components::flex::*, layout_components::*, ui_node::*,
```

Comment:
@alice-i-cecile I seem to have issues with not being able to pick up the `layout_components::flex::*` items in the examples. Any ideas on what can be done to resolve it?

This is the error I'm seeing:
> failed to resolve: use of undeclared type `AlignSelf`
use of undeclared type `AlignSelf`

Reply:
Does it work if you import it "explicitly", i.e. not from the prelude or is that also broken?

Reply:
It works if I use it as `flex::AlignSelf::Center` while having `use bevy::prelude::*;`

Reply:
Found the issue: During rebase I had accidentally kept 2x sets of these components which confused both the compiler and myself.